src/PasswordField.js

```diff
@@ -20,15 +20,12 @@ const getStyles = (props, context, state) => {
 
   const styles = {
     root: {
-      display: 'flex',
-      width: props.fullWidth ? '100%' : null
-    },
-    inputContainer: {
-      flex: 1
+      position: 'relative',
```

Comment:
I don't see it as too big of a problem. We could also wrap it with a static positioned element if need be.